### DISCUSSION TITLE
アイコンができません！

### DISCUSSION THREAD

**Discussion by KOTARO1209** [lit-iphone-course/lit-iphone-course#137] [Question-original] [answered]
## (1) 実現したいこと
アプリアイコンの見た目を変えたい
## (2) 実現するために自分が試した内容とその結果
アイコン教科書を見ながら試しました
## (3) 調べた内容や資料
pngの画像を使っているか確認しました。アプリを一度削除して再度実行しました
## (4) 原因だと思われそうな箇所(ソースコードをコピペして貼る)
不明
## (5) GitHubのリンク
https://github.com/KOTARO1209/icon

**Comment by txxxxc** [comment]
手元で試してみたんだけど、秒でicon表示されてしまった、、、
下のzip解答して実行してもアイコン表示されないか確認してもらっても良い...?
お願いします🙇‍♂️
[Sample.zip](https://github.com/lit-iphone-course/lit-iphone-course/files/10715386/Sample.zip)

**Comment by txxxxc** [comment, marked as answer]
ちなみに
```
Name: icon.png
Type: Default - PNG Image
```
になってる？
![image](https://user-images.githubusercontent.com/22112440/218295876-69e69516-5c2b-4218-9941-f461992693ff.png)

**Reply by KOTARO1209** [reply]
バージョンの違い？で開けなさそうです、、
<img width="285" alt="image" src="https://user-images.githubusercontent.com/75009903/218296320-b5f71ee0-8d14-4c7c-940a-41c821cbfb52.png">

なってると思います。
<img width="721" alt="image" src="https://user-images.githubusercontent.com/75009903/218296293-c417d644-e67a-46ef-a3ce-88f1d652cb1c.png">

**Reply by txxxxc** [reply]
んーーーー
一旦プロジェクト作り直してやってもらっても良い？

**Reply by KOTARO1209** [reply]
全て作り直してやったらできました🙇‍♂️

**Reply by txxxxc** [reply]
良かった！！笑
